ugo_collector/RatePlan_Created/RatePlan_Created_1_firstAnalysis.ipynb

```diff
@@ -0,0 +1,2472 @@
+      "text/plain": [
+       "<Figure size 1296x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(18,8))\n",
+    "sns.set(font_scale = 1.5)\n",
+    "sns.countplot(x = OTA_HotelRateAmountNotifRQ.CmRatePlanId, data = OTA_HotelRateAmountNotifRQ, hue = OTA_HotelRateAmountNotifRQ.Origin, palette = 'viridis')\n",
+    "plt.title(\"Breaking CmRatePlanId by Origin\")\n",
+    "plt.show()"
+   ]
```

Comment:
To prevent overlapping labels on `x` axis:
``` Python
fig, ax = plt.subplots(figsize = (18,8))
...
ax = ax.set_xticklabels(labels=x_dates, rotation=90, ha='right')
```

Reply:
@bluejurand thank you for underlying that, you are right, I missed it